src/main/scala/nl/knaw/dans/easy/solr/EasyUpdateSolrIndex.scala

```diff
@@ -35,7 +35,7 @@ object EasyUpdateSolrIndex {
 
   /** API for EasyIngestFlow, assumes a single dataset id, fails on any type of error */
   def run(implicit settings: Settings): Try[Unit] = Try {
-      settings.solr.update(createSolrDoc(settings.datasets.head))
+      settings.solr.update("<add>"+createSolrDoc(settings.datasets.head)+"</add>")
```

Comment:
The Scala interpolated string syntax is nicer:

      s"<add>${createSolrDoc(settings.datasets.head)}</add>"